pilot5-submission/pilot5-programs/tlf-efficacy.r

```diff
@@ -43,27 +34,27 @@ itt <- adsl[adsl[["ITTFL"]] == "Y", c("STUDYID", "USUBJID")]
 adlb1 <- adlb %>%
   dplyr::right_join(itt, by = c("STUDYID", "USUBJID")) %>%
   subset(TRTPN %in% c(0, 81) & PARAMCD == "GLUC" & !is.na(AVISITN)) %>%
-  mutate(TRTPN = ifelse(TRTPN == 0, 99, TRTPN)) # change treatment order for pairwise comparison
+  dplyr::mutate(TRTPN = ifelse(TRTPN == 0, 99, TRTPN)) # change treatment order for pairwise comparison
 
 ## Fit data for linear model
 gluc_lmfit <- adlb1 %>%
-  filter(AVISITN == 20) %>%
-  lm(CHG ~ BASE + TRTPN, data = .)
+  dplyr::filter(AVISITN == 20) %>%
```

Comment:
@SHAESEN2  can you drop the use of `pkg_name::` from the scripts unless there is a namespace conflict.